src/app/store/courses/courses.facade.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { CoursesSelectors } from './courses.selectors';
+import { CoursesActions } from './courses.actions';
+import { Course } from 'src/app/features/courses/courses-model';
+
+@Injectable()
+export class CoursesStateFacade {
+  isCoursesLoading$ = this.store.select(CoursesSelectors.selectIsCoursesLoading);
+  isCourseLoading$ = this.store.select(CoursesSelectors.selectIsCourseLoading);
+  isSearchingState$ = this.store.select(CoursesSelectors.selectIsSearchingState);
```

Comment:
Why do we need this unused variable `isSearchingState$` and `errorMessage$`?

Reply:
have removed